### PR TITLE
(#628) Added io.netty.allocator.numDirectArenas=0 flag to jvm.options

### PR DESCRIPTION
**Pull request by andrx**
(#628) Added io.netty.allocator.numDirectArenas=0 flag to overcome RAM issues on ES >= 7.4.0

### CODE REVIEW

**Comment by jmlrt** [comment]
jenkins test this please

File: templates/jvm.options.j2 [hunk 1]
```diff
@@ -67,6 +67,9 @@
 -Dio.netty.noUnsafe=true
 -Dio.netty.noKeySetOptimization=true
 -Dio.netty.recycler.maxCapacityPerThread=0
+{% if es_version | version_compare('7.4.0', '>=')  %}
```
Comment (jmlrt) [inline templates/jvm.options.j2]:
```suggestion
{% if es_version is version('7.4.0', '>=')  %}
```

`{% if es_version | version_compare('7.4.0', '>=')  %}` is working for our ubuntu-1804 tests which are using Ansible 2.5.1 but is failing for other tests which are using newer version of Ansible.

According to [Ansible doc](https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#version-comparison), the recommended syntax is now `{% if es_version is version('7.4.0', '>=')  %}`

Comment (andrx) [reply templates/jvm.options.j2]: $ yum list ansible.noarch
	Loaded plugins: fastestmirror
	Loading mirror speeds from cached hostfile
	 * base: mirror.vcu.edu
	 * epel: iad.mirror.rackspace.com
	 * extras: mirror.vcu.edu
	 * updates: mirror.vcu.edu
	Installed Packages
	ansible.noarch		2.8.1-1.el7		@epel
	Available Packages
	ansible.noarch		2.9.1-1.el7		epel

So, I tested with 2.8.1.
Okay, thanks!

**Comment by jmlrt** [comment]
jenkins test this please

**Comment by jmlrt** [comment]
@andrx I was ready to merge your PR but it seems that jvm options will change in next Elasticsearch release https://github.com/elastic/elasticsearch/pull/48252

**Comment by andrx** [comment]
@jmlrt hey, no problem... great to hear.
so, will you update the template accordingly or I misunderstood how it's going to work

**Comment by jmlrt** [comment]
Yeah I'll need to check in details what is required

If there is just things to change in the current `jvm.options` template, maybe I'll be able to post changes required to this PR so you can include them before merging.

If a different solution is required (maybe use default `jvm.options` file from Elasticsearch package and allow to fully overwrite it only if some variables are provided to ansible) I'll create a new PR and close this one.

Anyway thank you for your work on this PR 👍

**Comment by andrx** [comment]
Sure, great that it will be sorted with ES 7.5

**Comment by jmlrt** [comment]
@andrx, I was finally able to merge https://github.com/elastic/ansible-elasticsearch/pull/637 which handle jvm options. I'm closing your PR. Thank you for your work here👍

**Comment by andrx** [comment]
@jmlrt just curious when do you plan to release those. at the moment i temporarily pointed it to master but would like to set a specific version.
thanks in advance

**Comment by jmlrt** [comment]
We were waiting #637 merge to create 7.5.0 release and may still have a couple of small pr to pass, so the release should come in the beginning of next week.

**Comment by andrx** [comment]
got it. thank you!

**Pull request closed without merging**